lib/core/plugin/pluginContext.ts

```diff
@@ -206,6 +206,12 @@ export class PluginContext {
    */
   public errorsManager: any;
 
+  /**
+   * Current Kuzzle node unique identifier
+   */
+  public nodeId: string;
+
+
```

Comment:
I hesitate to place it in the accessors (PluginContext.accessors) rather than as a property of the PluginContext class. I'm open to discussion 🙂

Reply:
I agree, it should be placed in `accessors` to comply with actual structure